woocommerce/woocommerce-gutenberg-products-block
# Move `Textarea` to components package

## What

Move `Textarea` component to components package and update the place it is imported:

- `OrdeNotes`

## Why


As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add an item to your cart.
2. Go to the Checkout block, ensure the order notes section displays correctly, check the option to add a note to your order and ensure that the text area appears.
3. Enter text into it and check out.
4. Ensure your note is saved alongside your order.
<img width="529" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/52fa2815-8c37-4cf6-b6fe-1b487fe6eefd">



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `Textarea` component available in the `@woocommerce/blocks-components` package.
